### DISCUSSION TITLE
GitHub says fork exists when it really doesn’t

### DISCUSSION THREAD

**Discussion by wjk** [community/community#130668] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I am trying to fork the [trunkmaster/apple-corefoundation](https://github.com/trunkmaster/apple-corefoundation) repository into my personal account. I go to the repository, click the Fork button, and then I am prompted to select an owner. It says that I already have a fork of this repository under @wjk, and as such is refusing to let me continue. But there is no fork of this repository under @wjk; I have never forked this repository before. Can anyone help me?

**Comment by davevad93** [comment, marked as answer]
Hi @wjk ,

The problem you are experiencing is because you already have a fork of the original [apple/swift-corelibs-foundation](https://github.com/apple/swift-corelibs-foundation) repo and [trunkmaster/apple-corefoundation](https://github.com/trunkmaster/apple-corefoundation) is a fork of that same repo, so you basically can't fork it again unless you delete [wjk/swift-corelibs-foundation](https://github.com/wjk/swift-corelibs-foundation). GitHub doesn't allow forking an already forked repository.

**Reply by wjk** [reply]
Thanks!

**Reply by davevad93** [reply]
You're welcome!
